src/Aspire.Dashboard/Components/Controls/LogViewer.razor.cs

```diff
@@ -104,6 +103,7 @@ internal async Task ClearLogsAsync()
     public async ValueTask DisposeAsync()
     {
         await _cancellationSeries.ClearAsync();
+        ViewModel.LogEntries.Clear();
```

Comment:
Could you add a TODO here that this can be removed when https://github.com/dotnet/aspire/issues/4961 is done.

I think the solution here will be temporary and long term we can make this dependency transient so it doesn't matter if there is left over state.

Reply:
@kvenkatrajan asked me to merge, so I went ahead and did this